doc/api/packages.md

```diff
@@ -405,7 +442,6 @@ analogous to the exports field.
 <!-- YAML
 added:
   - v14.13.0
-  - v12.20.0
```

Comment:
Why was that line deleted?

Reply:
As alluded to in the original description, if we are using pattern trailers in the examples, or other later features, while stating version support to 12, it gets quite confusing to separate the specific exports features support, and we want to encourage pattern trailers mappings (`"./*.js": "./*.js"`) in examples over without (`"./*": "./*.js"` or `"./*": "./*"`) for the reasons around file extension handling.

Reply:
Are we able to include description metadata with these version items?

Reply:
Maybe like this? https://github.com/nodejs/node/blob/611847bd88ae2af952e0bc5b810edf1765452c71/doc/api/esm.md?plain=1#L326-L331

If you want to avoid 12.20.0 appearing in the generated HTML, maybe mention it in a `description` line for 14.13.0.

Reply:
The way I think about it is this - patterns were experimental for a while and went through some changes, and the first non-buggy implementations should be considered the stable baseline.

And simply, Node.js 12 is not a stable patterns target because most shouldn't need to know the difference between the pattern support granularity here.

I'd rather a user doesn't use patterns to support Node.js 12, than contort to avoiding pattern trailers as somehow unstable, that's the issue this is trying to address.

Reply:
Yeah I suppose so.

Ok, instead I've included the extra change items here detailing when imports support for patterns was added, and when pattern trailers support was added. The example uses both of these features but since we're now explicitly showing this history, the stable baseline is clear.